backend/apps/volontulo/views/api.py

```diff
@@ -98,3 +102,21 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     queryset = models.Organization.objects.all()
     serializer_class = serializers.OrganizationSerializer
     permission_classes = (permissions.OrganizationPermission,)
+
+    @detail_route(methods=['POST'], permission_classes=(AllowAny,))
```

Comment:
I would prefer to change `OrganizationPermission` in `backend/apps/volontulo/permissions.py` to allow any instead of hardcoding it here.